tcf_website/migrations/0003_auto_20210330_2024.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.1.7 on 2021-03-30 20:24
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tcf_website', '0002_auto_20210313_1721'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='coursegrade',
+            old_name='not_pass',
+            new_name='no_credit',
+        ),
+        migrations.RenameField(
+            model_name='courseinstructorgrade',
+            old_name='not_pass',
+            new_name='no_credit',
+        ),
+    ]
```

Comment:
Will do! - it ended up being like this because the field got renamed after I had made the original migration files, and I never got around to changing it.